openra/handlers.py

```diff
@@ -78,7 +78,7 @@ def ProcessUploading(self, user_id, f, post, rev=1, pre_r=0):
             if not self.LegacyImport(tempname, parser):
                 misc.send_email_to_admin_OnMapFail(tempname)
                 return 'Failed to import legacy map.'
-            shutil.move(parser + "/" + self.legacy_name, tempname)
+            shutil.move(self.legacy_name, tempname)
```

Comment:
Was removing the `parser + "/"` intentional?

Reply:
Yes, I think a previous command dumps a file out at the current directory. Previously where it changed to the parser directory before running the command, that would have been the location of the file. Now it's just the base directory.

This isn't really much more likely to cause issues now than before, as dumping files out in the parser directory isn't ideal either.

I know this isn't great and should be looked into, but maybe this should be raised as a separate issue in itself (to make it place the file into a temporary directory instead). This way it can be given proper attention, rather than be a tiny thing in a big PR.